src/tools/runk/libcontainer/src/container.rs

```diff
@@ -358,6 +364,35 @@ pub fn create_linux_container(
     Ok(container)
 }
 
+// Load rustjail's Linux container.
+// "uid_map_path" and "gid_map_path" are always empty, so they are not set.
+pub fn load_linux_container(
+    container: &Container,
+    console_socket: Option<PathBuf>,
+    logger: &Logger,
+) -> Result<LinuxContainer> {
+    let status = &container.status;
+    let mut container = LinuxContainer::new(
```

Comment:
Nit: this `container` variable name is the same as the input parameter.

Reply:
Thanks. I noticed that `load_linux_container` does not need a whole `container` object, so I replaced it with `status::Status`.